internal/i18n/ru.json

```
@@ -322,6 +322,7 @@
     "name": "Имя",
     "key": "Ключ",
     "the_value": "Значение",
+    "description": "Описание",
```

Biome (2.5.5)

[error] The key description was already declared.

(lint/suspicious/noDuplicateObjectKeys)